### ISSUE TITLE
Feedback in Visual Studio Code, but not from tslint CLI

### ISSUE THREAD

**Issue by akrueger** [mgechev/codelyzer#309]
```
@angular/core": "^4.1.2
codelyzer": "^3.0.1
tslint": "^5.2.0
```

I am getting feedback in Visual Studio Code, but nothing when I run tslint via package.json.

`tslint .` comes back clean, but I have many rules being applied in VSC.

**Comment by mgechev** [comment]
That's matter of how you run the tool. I don't see provising any configuration over CLI. For more details visit tslint's documentation and make sure you invoke the linter with proper configuration and target.

**Comment by akrueger** [comment]
I have fixed the linter config target and am now receiving:

```
Could not find implementations for the following rules specified in the configuration:
    directive-selector
    component-selector
    use-input-property-decorator
    use-output-property-decorator
    use-host-property-decorator
    no-attribute-parameter-decorator
    no-input-rename
    no-output-rename
    no-forward-ref
    use-life-cycle-interface
    use-pipe-transform-interface
    pipe-naming
    component-class-suffix
    directive-class-suffix
    templates-use-public
    no-access-missing-member
    invoke-injectable
    template-to-ng-template
Try upgrading TSLint and/or ensuring that you have all necessary custom rules installed.
If TSLint was recently upgraded, you may have old rules configured which need to be cleaned up.

No valid rules have been specified
```

I have the following `tslint.json`:

```
{
  "extends": [ "codelyzer" ],
  "rules":{
    "directive-selector": [true, "attribute", "sg", "camelCase"],
    "component-selector": [true, "element", "sg", "kebab-case"],
    "use-input-property-decorator": true,
    "use-output-property-decorator": true,
    "use-host-property-decorator": true,
    "use-view-encapsulation": true,
    "no-attribute-parameter-decorator": true,
    "no-input-rename": true,
    "no-output-rename": true,
    "no-forward-ref": true,
    "use-life-cycle-interface": true,
    "use-pipe-transform-interface": true,
    "pipe-naming": [true, "camelCase", "sg"],
    "component-class-suffix": true,
    "directive-class-suffix": true,
    "templates-use-public": true,
    "no-access-missing-member": true,
    "invoke-injectable": true,
    "template-to-ng-template": true
  }
}
```

My `package.json` script is:

`"lint": "tslint './src_client/**/*.ts'",`

**Comment by mgechev** [comment]
Follow the [instructions here](https://github.com/mgechev/codelyzer#how-to-use).

**Comment by akrueger** [comment]
Yes, I followed [these](https://github.com/mgechev/codelyzer#custom-setup). At the initial step however I installed the latest of the packages rather than the specific ones here:

`npm i codelyzer@~2.0.0 tslint@4.0.0 typescript@2.0.9 @angular/core@2.4.0 @angular/compiler@2.4.0 rxjs@5.0.1 zone.js@0.7.2`

It's not clear that those are the required versions as the project itself has `3.0` and `4.0` with no mention of requirements for using older angular or tslint packages.

I then put the supplied `tslint.json` in my project per this step: 
`A. Using codelyzer package in PATH`

And I am using the 2nd option from below:
```
To run TSLint with this setup you can use one of the following alternatives:

1) Install codelyzer globally npm install -g codelyzer

2) Run TSLint from a package.json script by adding a script like tslint . to your package.json, similar to:

"scripts": {
  ...
  "lint": "tslint .",
  ...
},

```
